src/rust/en.reaperscans/res/filters.json

```
@@ -1,5 +1,82 @@
 [
 	{
 		"type": "title"
+	},
+	{
+		"type": "group",
+		"name": "Genres",
+		"filters": [
+			{
+				"type": "genre",
+				"id": "1",
+				"name": "Action",
+				"canExclude": false
+			},
+			{
+				"type": "genre",
+				"id": "2",
+				"name": "Fantasy",
+				"canExclude": false
+			},
+			{
+				"type": "genre",
+				"id": "3",
+				"name": "Romance",
+				"canExclude": false
+			},
+			{
+				"type": "genre",
+				"id": "4",
+				"name": "Martial Arts",
+				"canExclude": false
+			},
+			{
+				"type": "genre",
+				"id": "5",
+				"name": "Gaming",
+				"canExclude": false
+			},
+			{
+				"type": "genre",
+				"id": "6",
+				"name": "Comedy",
+				"canExclude": false
+			},
+			{
+				"type": "genre",
+				"id": "7",
+				"name": "Horror",
+				"canExclude": false
+			},
+			{
+				"type": "genre",
+				"id": "8",
+				"name": "Drama",
+				"canExclude": false
+			}
+		]
+	},
+	{
+		"type": "select",
+		"name": "Status",
+		"options": [
+			"All",
+			"Ongoing",
+			"Hiatus",
+			"Dropped",
+			"Completed"
+		],
+		"default": 0
+	},
+	{
+		"type": "select",
+		"name": "Order By",
+		"options": [
+			"Last Updated",
+			"Popular",
+			"Alphabetical",
+			"Oldest"
+		],
+		"default": 0
 	}
-]
+]
```

make sure files have newlines
```suggestion
]

```